src/lib/features/feature-toggle/feature-toggle-service.ts

```diff
@@ -1182,6 +1205,27 @@ export class FeatureToggleService {
         }
     }
 
+    private async addReleasePlansToEnvironments(
```

Comment:
this is the least invasive way to add release plans. For non flag enabled clients it does nothing